### PR TITLE
Organise packaging

### PR DESCRIPTION
**Pull request by mortenlj**
Moved files related to packaging to dedicated subdirectory, and adjusted relevant build scripts.

### CODE REVIEW

**Comment by sechmann** [comment]
Nice

But please, `s/macosx/macos/g`

File: Makefile [hunk 1]
```diff
@@ -12,6 +12,12 @@ dev-apiserver: local-postgres local-apiserver stop-postgres
 integration-test: stop-postgres-test run-postgres-test run-integration-test stop-postgres-test
 clients: linux-client macos-client windows-client
 
+# Ensures build tools are installed on linux
+.linux-init:
+	sudo apt update
+	sudo apt-get install --yes build-essential libgtk-3-dev libappindicator3-dev
```
Comment (x10an14) [inline Makefile]: Trenger vi faktisk `--yes` her...? Den funker fint interaktivt kjørt av `make`... kan alltids legge `shell` foran om man er usikker på det vel?

**Comment by mortenlj** [comment]
> But please, `s/macosx/macos/g`

:facepalm:

**Review by x10an14 — COMMENTED** [review]
I don't think having "automatic install in the background - user must pay attention and abort (read: press ctrl+c in time) to stop if these packages installed in the background are unwanted/unnecessary".

I find it perfectly plausible that in the not-too-distant futures (months at most) another different package might replace the functionality of `libappindicator3-dev` (for example).

**Review by x10an14 — APPROVED** [review]
LGTM! =)

**Pull request merged**